server/controllers/assessmentDayDataController/index.js

```diff
@@ -0,0 +1,66 @@
+import AssessmentDayDataModel from '../../models/AssessmentDayDataModel'
+import DataFileModel from '../../models/DataFileModel'
+import AssessDayDataService from '../../services/AssessmentDayDataService'
+import _ from 'lodash'
+
+const AssessmentDayDataController = {
+  create: async (req, res) => {
+    try {
+      if (_.isEmpty(req.body))
```

Comment:
Where do we stand on Lodash as a dependency? The last release was in 2016: https://github.com/lodash/lodash. I would prefer to move away from it if possible.

Reply:
can you help me understand why you would prefer to move away from it..?

Reply:
Yeah we can only import what we are using in the code for now to reduce package load.
It's all native javascript and no outside dependencies so I'm not entirely convinced that it can be that harmful.
Yeah as a far as utilities go they could be updated to be made faster, anyway i've refactored it using es6.
[cbc2b48](https://github.com/AMP-SCZ/dpdash/pull/382/commits/cbc2b48e0dc22a1dfc9c3e6ac53496fc5c7e90c1)